overrides/windows-override.json

```diff
@@ -76,6 +76,9 @@
                 },
                 "freemium": {
                     "state": "disabled"
+                },
+                "remoteBrokerDelivery": {
+                    "state": "disabled"
```

Comment:
Adding this satisfies the "All subfeatures must be defined in their overrides files if they apply” CI check.

Not sure it’s the right way to do it, as technically Windows supports remote broker delivery but it’s not behind any feature flag.

cc @jonathanKingston

Reply:
I think making it "disabled" makes the least confusion for it's current state.

We could just make it macos specific? So not defined in the feature file?

Reply:
We already have these two features:
- https://github.com/duckduckgo/privacy-configuration/blob/main/features/broker-protection.json: used in C-S-S
- https://github.com/duckduckgo/privacy-configuration/blob/main/features/dbp.json: used in Windows/macOS for waitlist
Can we add this as a sub-feature to the latter?

Reply:
I already did, but CI wants it to be in windows-override.json too https://github.com/duckduckgo/privacy-configuration/blob/634b7e2c5de6d3768ded853329b253b72cea9f43/features/dbp.json#L16-L18

Reply:
In this case I agree with @jonathanKingston, defining it only in `macos-override.json` would be the least confusing if it's possible.

Reply:
Sounds good, I’ve removed these other changes. @jonathanKingston Back to you for final approval (I think).

Reply:
Now it's only a change in `macos-override.json`, this PR can be approved by any Apple dev.